Example_13 test is broken

It seems that we broke example_13 (or at least the test) with https://github.com/ros-controls/ros2_control/pull/2347

```
     [ros2_control_node-1] [INFO] [1753760884.651091427] [controller_manager]: Activating controllers: [ threedofbot_joint_state_broadcaster threedofbot_pid_gain_controller ]
    [ros2_control_node-1] [WARN] [1753760884.651350703] [controller_manager]: Component 'FakeThreeDofBot' is in INACTIVE state, but has start interfaces to switch: 
    [NON-XML-CHAR-0x1B][91mCould not switch controllers since prepare command mode switch was rejected.[NON-XML-CHAR-0x1B][0m
    [ros2_control_node-1] Start interfaces: 
    [ros2_control_node-1] [
    [ros2_control_node-1]   threedofbot_joint1/pid_gain
    [ros2_control_node-1]   threedofbot_joint2/pid_gain
    [ros2_control_node-1]   threedofbot_joint3/pid_gain
    [ros2_control_node-1] ]
    [ros2_control_node-1] Stop interfaces: 
    [ros2_control_node-1] [
    [ros2_control_node-1] ]
    [ros2_control_node-1] 
Error: s2_control_node-1] [ERROR] [1753760884.651373715] [controller_manager]: Could not switch controllers since prepare command mode switch was rejected.
```


https://github.com/ros-controls/ros2_control_demos/issues/859
https://github.com/ros-controls/ros2_control_demos/issues/861
https://github.com/ros-controls/ros2_control_demos/issues/862
https://github.com/ros-controls/ros2_control_demos/issues/863
https://github.com/ros-controls/ros2_control_ci/issues/421
https://github.com/ros-controls/ros2_control_ci/issues/422
https://github.com/ros-controls/ros2_control_ci/issues/423
https://github.com/ros-controls/ros2_control_ci/issues/424
https://github.com/ros-controls/ros2_control_ci/issues/425

--- a/example_13/doc/userdoc.rst
+++ b/example_13/doc/userdoc.rst
@@ -255,12 +255,12 @@ Tutorial steps
     threedofbot_position_controller[forward_command_controller/ForwardCommandController] inactive
 
 
-3. Configure ``FakeThreeDofBot`` and its joint state broadcaster and non-movement command interfaces. Call
+3. Configure ``FakeThreeDofBot`` and its joint state broadcaster. Call
 
   .. code-block:: shell
 
     ros2 control set_hardware_component_state FakeThreeDofBot inactive
-    ros2 control switch_controllers --activate threedofbot_joint_state_broadcaster threedofbot_pid_gain_controller
+    ros2 control switch_controllers --activate threedofbot_joint_state_broadcaster
 
   Scenario state:
 
@@ -280,14 +280,14 @@ Tutorial steps
           state: id=2 label=inactive
           command interfaces
                   threedofbot_joint1/position [available] [unclaimed]
-                  threedofbot_joint1/pid_gain [available] [claimed]
+                  threedofbot_joint1/pid_gain [available] [unclaimed]
                   threedofbot_joint2/position [available] [unclaimed]
-                  threedofbot_joint2/pid_gain [available] [claimed]
+                  threedofbot_joint2/pid_gain [available] [unclaimed]
                   threedofbot_joint3/position [available] [unclaimed]
-                  threedofbot_joint3/pid_gain [available] [claimed]
+                  threedofbot_joint3/pid_gain [available] [unclaimed]
     ...
 
-  Controllers status, ``threedofbot_joint_state_broadcaster`` and ``threedofbot_pid_gain_controller`` are in active state now:
+  Controllers status, ``threedofbot_joint_state_broadcaster`` is in active state now:
 
   .. code-block:: shell
 
@@ -300,7 +300,7 @@ Tutorial steps
     rrbot_with_sensor_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
     rrbot_with_sensor_position_controller[forward_command_controller/ForwardCommandController] active
     threedofbot_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
-    threedofbot_pid_gain_controller[forward_command_controller/ForwardCommandController] active
+    threedofbot_pid_gain_controller[forward_command_controller/ForwardCommandController] inactive
     threedofbot_position_controller[forward_command_controller/ForwardCommandController] inactive
 
 
@@ -329,12 +329,12 @@ Tutorial steps
   - left robot is moving
   - middle robot is now still "standing"
 
-5. Activate ``FakeThreeDofBot`` and its controller. Call
+5. Activate ``FakeThreeDofBot`` and its controllers. Call
 
   .. code-block:: shell
 
     ros2 control set_hardware_component_state FakeThreeDofBot active
-    ros2 control switch_controllers --activate threedofbot_position_controller
+    ros2 control switch_controllers --activate threedofbot_pid_gain_controller threedofbot_position_controller
 
   Scenario state:
 

--- a/example_13/test/test_three_robots_launch.py
+++ b/example_13/test/test_three_robots_launch.py
@@ -172,7 +172,6 @@ class TestFixture(unittest.TestCase):
                 "switch_controllers",
                 "--activate",
                 "threedofbot_joint_state_broadcaster",
-                "threedofbot_pid_gain_controller",
             ]
         )
         cnames = [
@@ -184,7 +183,6 @@ class TestFixture(unittest.TestCase):
             "rrbot_with_sensor_fts_broadcaster",
             "rrbot_with_sensor_position_controller",
             "threedofbot_joint_state_broadcaster",
-            "threedofbot_pid_gain_controller",
         ]
         check_controllers_running(self.node, cnames)
         # still the same joint_states
@@ -248,6 +246,7 @@ class TestFixture(unittest.TestCase):
                 "switch_controllers",
                 "--activate",
                 "threedofbot_position_controller",
+                "threedofbot_pid_gain_controller",
             ]
         )
         cnames = [

